styles/styles.css

```diff
@@ -26,8 +26,14 @@
   --color-gray-light: #ccc;
   --color-gray-lighter: #f2f2f2;
   --color-white: #fff;
-
-  /* colors assignments */
+  --color-navy: #003B75;
+  --color-green: #41AD49;
+  --color-yellow: #FFF200;
+  --color-orange: #EE9D38;
+  --color-red: #D2232A;
```

Comment:
this red has insufficient color contrast (as a background color to the text color #003360) for accessibility if that is a concern.

Reply:
that's usually how it goes! just thought i'd mention.